packages/components/src/components/Dropzone/Dropzone.tsx

```diff
@@ -0,0 +1,252 @@
+  borderColor?: SystemProp<keyof Theme["colors"], Theme>;
+} => {
+  if (status === "loading") {
+    return {
+      backgroundColor: "colorBackgroundInfo",
+      borderColor: "colorBorderPrimary",
+    };
+  }
+  if (status === "success") {
+    return {
+      backgroundColor: "colorBackgroundSuccess",
+      borderColor: "colorBorderSuccess",
+    };
+  }
+  if (isDragAccept) {
+    return {
+      backgroundColor: "colorBackgroundInfo",
+      borderColor: "colorBorderPrimary",
+    };
+  }
+  if (isDragReject || status === "error" || status === "uploadError") {
+    return {
+      backgroundColor: "colorBackgroundError",
+      borderColor: "colorBorderError",
+    };
+  }
+  if (isFocused) {
+    return {
+      backgroundColor: "colorBackground",
+      borderColor: "colorBorderPrimary",
+    };
+  }
+  return {
+    backgroundColor: "colorBackgroundWeakest",
+    borderColor: "colorBorder",
+  };
+};
+
+const getIcon = (
+  status: string,
+  isDragActive: boolean,
+  isDragAccept: boolean,
+  isDragReject: boolean
+) => {
+  let icon: IconProps["icon"] = "CloudArrowUpIcon";
+  let iconColor: IconProps["color"] = "colorIconInfo";
+  if ((!isDragActive && !isDragReject) || status === "success") {
+    icon = "CheckCircleIcon";
+    iconColor = "colorIconSuccess";
+  }
+  if (isDragAccept || status === "default" || status === "loading") {
+    icon = "CloudArrowUpIcon";
+    iconColor = "colorIconInfo";
+  }
+  if (isDragReject || status === "error" || status === "uploadError") {
+    icon = "ExclamationTriangleIcon";
+    iconColor = "colorIconError";
+  }
+
+  return <Icon color={iconColor} decorative icon={icon} size={"sizeIcon40"} />;
+};
+
+const Dropzone = React.forwardRef<HTMLDivElement, DropzoneProps>(
+  (
+    {
+      fileTypes = { "application/pdf": [".pdf"] },
+      cancelDocumentUpload,
+      sendDocument,
+    },
+    ref
+  ) => {
+    const [status, setStatus] = useState("default");
+    const [progress, setProgress] = useState(0);
+    const [errors, setErrors] = useState("");
+    const {
+      acceptedFiles,
+      getRootProps,
+      getInputProps,
+      isDragActive,
+      isDragAccept,
+      isDragReject,
+      isFocused,
+    } = useDropzone({
+      maxFiles: 1,
+      maxSize: MAX_FILE_SIZE,
+      multiple: false,
+      accept: {
+        ...fileTypes,
+      },
+      onDropAccepted: () => {
+        setStatus("loading");
+        sendDocument(acceptedFiles[0], (progress) => {
+          setProgress(progress);
+        })
+          .then(() => {
+            setProgress(0);
+            setStatus("success");
+          })
+          .catch(() => {
+            setStatus("uploadError");
+          });
+      },
+      onDropRejected: (fileRejections) => {
+        setStatus("error");
+        forEach(fileRejections, (file) => {
+          forEach(file.errors, (err) => {
+            if (err.code === "file-too-large") {
+              setErrors("File must be less than 50mb.");
```

Comment:
As pluto is opensource, what do you think about having a default max file size and allowing engineers to set the max they want as an optional prop? So, if someone else uses it, the component is it'll reusable. And not fixed for our needs.